package.json

```diff
@@ -75,11 +75,11 @@
   },
   "main": "./lib/bson.cjs",
   "module": "./lib/bson.mjs",
-  "browser": "./lib/bson.mjs",
   "exports": {
-    "browser": "./lib/bson.mjs",
     "import": "./lib/bson.mjs",
-    "require": "./lib/bson.cjs"
+    "require": "./lib/bson.cjs",
+    "react-native": "./lib/bson.cjs",
```

Comment:
I'm not 100% sure on this, but I think the `react-native` property should be one level up, looking at [the docs for the Metro config](https://facebook.github.io/metro/docs/configuration/#resolvermainfields) and [the spec that it references](https://github.com/defunctzombie/package-browser-field-spec).

Was this manually tested in a React Native project?

Reply:
Yes, I made a new project with `react-native-cli` and confirmed that this works with the bundling that comes with that setup, not sure if that is Metro. The `"exports"` property is standardized for [nodejs](https://nodejs.org/dist/latest-v18.x/docs/api/packages.html#exports), [webpack](https://webpack.js.org/guides/package-exports/), and rollup, I suspect it may just not be explicitly mentioned there? 

We can get this into an alpha release and maybe you have projects with more diverse tooling worth testing to see if it gets pulled in correctly, I'll ping you on slack when its available